content/en/serverless/google_cloud_run/_index.md

```diff
@@ -2,43 +2,68 @@
+
+Google Cloud Run is a fully managed serverless platform for deploying and scaling container-based applications. Datadog
+provides monitoring and log collection for Cloud Run through the [GCP integration][1]. Datadog also provides a solution,
+now in public beta, for instrumenting your Cloud Run applications with a purpose-built Agent to enable tracing, custom
+metrics, and direct log collection.
 
   <div class="alert alert-warning">This feature is in public beta. You can provide feedback through a <a href="https://forms.gle/HSiDGnTPvDvbzDAQA">feedback form</a>, or through your standard support channels. During the beta period, Cloud Run monitoring and APM tracing are available without a direct cost. Existing APM customers may incur increased span ingestion and volume costs. </div>
 
-## Tracing and custom metrics
-### Build your container
+## Getting started
+
+### Prerequisites
+
+Make sure you have a [Datadog API Key][10] and are using a programming
+language [supported by a Datadog tracing library][2].
+
+### Deploy a sample application in one click
+
+To deploy a sample application without the need to follow the rest of the guide, you can use
+the [Cloud Run Button](https://github.com/GoogleCloudPlatform/cloud-run-button)
+from one of [these examples][12]. The installation process will ask the required
+details.
+
+### Build your container with Datadog instrumentation
 
-If you are using a Dockerfile to build your application, complete the following:
+Cloud Run requires applications to be packaged in a container. From the application root folder follow the steps
+for either Dockerfile or buildpack build.
 
-1. Instrument your application with a [supported Datadog tracing library][2]
+#### Using Dockerfile
 
-2. Use the `COPY` instruction to copy the [Datadog `serverless-init` binary][3] into your Docker image.
+Using a Dockerfile to build your container, the following steps are needed:
 
-3. Use the `ENTRYPOINT` instruction to run the `serverless-init` binary as your Docker container is initiated.
+1. Use the `COPY` instruction to copy the [Datadog `serverless-init` binary][3] into your Docker image.
 
-4. Use the `CMD` instruction to run your existing application and other required commands as arguments.
+2. Use the `ENTRYPOINT` instruction to run the `serverless-init` binary as your Docker container is initiated.
 
-The following are examples of how to complete these three steps. You may need to adjust these examples depending on your existing Dockerfile setup. 
+3. Use the `CMD` instruction to run your existing application and other required commands as arguments.
 
+You can accomplish those steps by adding the following lines to your Dockerfile. You may need to adjust these examples
+depending on your existing Dockerfile setup.
 
 {{< programming-lang-wrapper langs="go,python,nodejs,java,dotnet,ruby" >}}
 {{< programming-lang lang="go" >}}
+
 ```
 COPY --from=datadog/serverless-init:beta4 /datadog-init /app/datadog-init
```

Comment:
Can you discuss with your team in standup about using `:latest` instead of say `beta4`? so we avoid updating the doc every release?

Reply:
I removed the tag from `serverless-init`and the buildpack.
The buildpack has no latest tag yet though, I will check on why

Reply:
ok we are going to push the `latest` tag to docker hub soon